src/test/timedata_tests.cpp

```diff
@@ -34,4 +34,22 @@ BOOST_AUTO_TEST_CASE(util_MedianFilter)
     BOOST_CHECK_EQUAL(filter.median(), 7);
 }
 
+
+BOOST_AUTO_TEST_CASE(util_MedianFilterShallNotGrowBeyondSize)
+{
+
+  CMedianFilter<int> filter(2,15);
```

Comment:
Run this through `clang-format` on the new code to make sure the formatting is consistent with the style guide :-)

Reply:
Also: please use four spaces for indentation

Reply:
See https://github.com/bitcoin/bitcoin/tree/master/contrib/devtools#clang-format-diffpy